### PR TITLE
fix: change pagination style

### PR DESCRIPTION
**Pull request by Exiest**

### CODE REVIEW

**Review by lazzystep — APPROVED** [review]

**Comment by vlacher12** [comment]
Replace pls default locales:
Show rows -> Items per page
1-12 of 201 items -> Displaying 1-12 of 201 items

**Comment by vlacher12** [comment]
Did you restyle TablePagination component?

File: src/components/pagination/table-pagination.tsx [hunk 1]
```diff
@@ -58,13 +70,21 @@ export const TablePagination = ({
   onChangeRowsPerPage,
   onChangePage,
   className,
+  selectStyles,
+  locales,
```
Comment (vlacher12) [inline src/components/pagination/table-pagination.tsx:74]: just use default 
```
  locales = {
    itemsPerPage: 'Items per page',
    displayingItems: (rangeStart, rangeEnd) => `Displaying ${rangeStart}-${rangeEnd} of ${count} items`,
    currentPage: (currentPage: number, pagesAmount: number) => `${currentPage + 1} of ${pagesAmount} pages`,
  },
```

File: src/components/pagination/table-pagination.tsx [hunk 2]
```diff
@@ -77,31 +97,29 @@ export const TablePagination = ({
           size={theme.typography.sizes.m}
           weight={theme.typography.weights.light}
           color={theme.colors.base900}
-          className="paginationTextBeforeSelect"
+          className="paginationText"
         >
-          {contentBeforeSelect}
+          {locales?.itemsPerPage || 'Items per page'}
```
Comment (vlacher12) [inline src/components/pagination/table-pagination.tsx:102]: and here you can put just `locales.itemsPerPage`

File: src/components/pagination/table-pagination.tsx [hunk 3]
```diff
@@ -77,31 +97,29 @@ export const TablePagination = ({
...
-          styles={additionStyles()}
+          styles={customSelectStyles(selectStyles)}
           isDisabled={disabled}
           onChange={rowsPerPageHandler}
           value={rowsPerPageOptions.find((option) => Number(option.value) === Number(rowsPerPage))}
         />
-        <div className={cx('paginationDivider', 'paginationDividerLeft')} />
         <Text size={theme.typography.sizes.m} color={theme.colors.base900} className="paginationText">
-          {`${count > 0 ? from : 0}-${to} of ${count} items`}
+          {locales?.displayingItems(rangeStart, rangeEnd, count) || displayingItemsLabel}
```
Comment (vlacher12) [inline src/components/pagination/table-pagination.tsx:116]: the same

File: src/components/pagination/table-pagination.tsx [hunk 4]
```diff
@@ -77,31 +97,29 @@ export const TablePagination = ({
...
         </Text>
       </div>
       <div className="paginationSection">
         <Text size={theme.typography.sizes.m} color={theme.colors.base900} className="paginationText">
-          {`${page + 1} of ${pagesAmount} pages`}
+          {locales?.currentPage(page + 1, pagesAmount) || currentPageLabel}
```
Comment (vlacher12) [inline src/components/pagination/table-pagination.tsx:121]: and the same

File: src/components/pagination/table-pagination.tsx [hunk 5]
```diff
@@ -29,27 +38,30 @@ const defaultOptions = [
   { label: '48', value: 48 },
 ];
 
-const additionStyles = () => ({
-  valueContainer: () => ({
+const customSelectStyles = (styles) => ({
+  valueContainer: (p, s) => ({
```
Comment (vlacher12) [inline src/components/pagination/table-pagination.tsx:42]: what is p and s?

File: src/components/pagination/styles.ts [hunk 6]
```diff
@@ -9,43 +9,42 @@ export const StyledPagination = styled.div<{
...
       text-decoration: none;
-      width: 28px;
+      width: 32px;
       border-bottom: none;
       background-color: transparent;
       cursor: pointer;
 
       &:hover {
         border-bottom: none;
+        background-color: ${(props: any): any => props.theme.colors.primary200};
```
Comment (vlacher12) [inline src/components/pagination/styles.ts:37]: i'd get theme from props:
`background-color: ${({ theme }): any => theme.colors.primary200};`

File: src/components/pagination/table-pagination.tsx (hunk 1 above)
Comment (Exiest) [reply src/components/pagination/table-pagination.tsx:74]: So should 'locales' be mandotory passed?

File: src/components/pagination/table-pagination.tsx (hunk 5 above)
Comment (Exiest) [reply src/components/pagination/table-pagination.tsx:42]: provided styles, and select's state. will rename this

File: src/components/pagination/table-pagination.tsx (hunk 1 above)
Comment (vlacher12) [reply src/components/pagination/table-pagination.tsx:74]: no, we can avoid passing `locales`

File: src/components/pagination/table-pagination.tsx [hunk 7]
```diff
@@ -29,27 +38,30 @@ const defaultOptions = [
...
+  control: (provided, state) => ({
+    minHeight: '48px',
     padding: '0 5px',
+    ...additionalStyles('control', styles, provided, state)
   }),
-  dropdownIndicator: () => ({
+  dropdownIndicator: (provided, state) => ({
     padding: 0,
     width: '16px',
+    ...additionalStyles('dropdownindicator', styles, provided, state)
```
Comment (vlacher12) [inline src/components/pagination/table-pagination.tsx:55]: why it isn't in camelCase?

**Review by vlacher12 — APPROVED** [review]

**Pull request merged**